src/AdScreenerTool.js

```diff
@@ -241,12 +260,34 @@ const AdScreener = (params) => {
       </p>
 
       <div className="App-filter-selector">
+      	<div>
```

Comment:
I think if you add `className="filter-selector"` to this div it will make the element the same size as the other dropdowns (due to flex auto sizing)

Reply:
This didn't help. The tab went to the next line. When I add one more search tab the size is actually fine, so I think it should look better once we add advertiser search.